src/atoms/buttons/button.tsx

```diff
@@ -332,7 +348,50 @@ const ButtonIconOnly = Styled(Button)`
     };
   `}
 `;
-export const ButtonAtom:FC<ButtonProps> = ({
+const SquareButton = Styled.button<GenericStylingProps>`
+  height: 70px;
```

Comment:
I'm not sure if width/height should be restricted to one size, maybe you can pass size prop that is optional, when not provided 70px is taken if it is replace the height and width with it (one prop so that it stays square).